catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -351,46 +352,48 @@ public Metacard transform(InputStream input, String id)
       Metadata metadata;
       String bodyText = null;
       String metadataText;
-      Metacard metacard;
-      String contentType;
+      Metacard metacard = new MetacardImpl(commonTikaMetacardType);
+      String contentType = DataType.DATASET.name();
       TikaMetadataExtractor extractor = null;
       try (InputStream inputStreamCopy = fileBackedOutputStream.asByteSource().openStream()) {
         extractor = new TikaMetadataExtractor(inputStreamCopy, previewMaxLength, metadataMaxLength);
 
-      } catch (TikaException e) {
-        throw new CatalogTransformerException(e);
+      } catch (Throwable t) {
```

Comment:
❗️ We should not blindly catch all throwables like that. We should first do a catch(VirtualMachineError) and throw that back without even attempting to log.

Reply:
Actually, why not just catch `RuntimeException` and any checked exceptions thrown by Tika?

Reply:
Definitely better! We should only resort to catching more when really required.

Reply:
Changed to catch checked exceptions and RuntimeException